content/en/blog/2022/2022-07-25-enhancing-kubernetes-one-kep-at-a-time/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: blog
+title: "Enhancing Kubernetes one KEP at a Time"
+date: 2022-07-25
```

Comment:
I'll take the liberty of bumping the publication date.
@rhockenbury, for this blog we usually _omit_ the publication date from the filename (it's not a strong convention). For a single-file article we usually wouldn't have that as a single Markdown file too - no directory.

We can merge this as-is and fix those details later as they have no impact on the published article.


```suggestion
date: 2022-08-11
```